_config/products.yml

```diff
@@ -124,6 +132,32 @@ devstudio:
         release_notes_url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_Developer_Studio/
         blog_announcement_url: /blog/12.0.0.ga-for-photon.html
         eclipse_name: Eclipse Photon 4.8.0
+        installers:
```

Comment:
The blog is still the old one, the Eclipse name is still Photon and the release date is July 2018.

Reply:
good catch. My copy-paste-edit skills are fail today.